plugin/src/main/groovy/com/novoda/gradle/command/Monkey.groovy

```diff
@@ -4,51 +4,24 @@ import org.gradle.api.tasks.TaskAction
 
 class Monkey extends AdbTask {
 
-    def events
-    def seed
-    def categories
-
     protected handleCommandOutput(def text) {
         super.handleCommandOutput(text)
-        if (text.contains("Monkey aborted")) {
+        if (text.toLowerCase().contains("monkey aborted")) {
```

Comment:
Here, this was kinda a `hack` to fail the build if the monkey is aborted. 
Monkey's output sometimes starts with upper case `Monkey`, sometimes starts with some other message that ends with lower case `monkey`.
So now it will fail whatever the case.